src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -22,77 +23,82 @@ install and update these dependencies for Axom.
-the individual packages build systems, as well as any packages they depend on.
+and configurations of software packages. It has recipes for building each package
+with available variants to customize it to your needs. For example, Axom
+has variants for Fortran, MPI, and others. The recipes drive
+the individual packages build systems and manage packages they depend on.
 Spack also handles system level packages, so you can describe where they are on your
-system instead of building them from scratch.  You will need to describe which compilers
+system instead of building them from scratch. You will need to describe which compilers
 are available on your system as well.
 
-* Platform specific configuration files live under ``scripts/spack/configs/<platform name>``.
-  There is one file (``spack.yaml``) per platform that handles the following:
+* Platform specific configuration files live under ``axom/scripts/spack/configs/<platform name>``.
+  There is one file (``spack.yaml``) per platform that has sections describing:
 
    * ``compilers``: This section contains the compiler specs that describe the location
-     and any other required information about that compiler.  For example, compiler or 
-     linker flags.
-   * ``packages``: This section describes the system level packages.  For example,
-     where they are located and what version they are. This file is very important
-     due to its ability to drastically reduce the amount of packages that Spack builds.
-
-* Axom specific Spack package files live under ``scripts/spack/packages``. These override
-  the package files that live in Spack's repository here ``var/spack/repos/builtin/packages``.
-  We try to minimize these but we have had to alter the existing packages to apply fixes before
-  pushing them up to Spack proper or alterations to the recipes that are Axom specific.
-  This overriding does not happen at the Spack level, but at the next level, Uberenv.
-* `Spack's GitHub repo <https://github.com/spack/spack>`_
-* `Spack's documentation <https://spack.readthedocs.io/en/latest/>`_
+     and any other required information for each compiler. For example, a compiler spec
+     contains compiler and version, build and linker flags, etc.
+   * ``packages``: This section describes system level packages. For example, version and
+     location on the filesystem. This file is very important
+     due to its ability to drastically reduce the number of packages that Spack builds.
+
+* Axom specific Spack package files live under ``axom/scripts/spack/packages``. These override
+  the package files in Spack's repository under ``var/spack/repos/builtin/packages``.
+  We try to minimize these, but we often have to alter the existing Spack packages to: apply fixes
+  before pushing them up to Spack proper, or alter recipes in ways that are Axom specific.
+  Such overrides do not happen at the Spack level, but at the next level, Uberenv 
+  described below.
```

Comment:
I suggest:
- remove the colon from "to: apply fixes"
- remove the comma from "up to Spack proper, or alter recipes"
- either add a comma, making it "the next level, Uberenv, described below," or add parentheses, making it "the next level, Uberenv (described below)."

The last point is needed because "described below" is an adjective phrase modifying "Uberenv."  Without the comma or the parens, overrides happen at the noun phrase "Uberenv described below" which makes less sense.

Reply:
Got it. Change made.